x/feegrant/fees.go

```diff
@@ -22,4 +24,7 @@ type FeeAllowanceI interface {
 	// ValidateBasic should evaluate this FeeAllowance for internal consistency.
 	// Don't allow negative amounts, or negative periods for example.
 	ValidateBasic() error
+
+	// ExpTime returns the expiry time of the allowance.
```

Comment:
`ExpTime` sounds odd, how about just `Expiry` or `Expiration`?

Reply:
We cannot add `Expiration()` since a field name with `Expiration` already exists, but yeah `Expiry` looks fine.

Reply:
How about `ExpiresAt`?